src/tplr/comms.py

```diff
@@ -442,7 +442,7 @@ async def s3_get_object(
                         loaded_data = torch.load(
                             temp_file_path,
                             map_location=self.config.device,
-                            weights_only=True,
+                            #weights_only=True,
```

Comment:
If you could just delete the line instead of commenting out to reduce some noise it would be great.